frontend/src/components/Network.tsx

```diff
@@ -62,6 +62,11 @@ import { lerp } from "@/util/math";
 import { formatNumber } from "@/util/string";
 import classes from "./Network.module.css";
 
+type Pan = {
```

Comment:
A small thing, but you could extract/derive this prop instead of redefining it. See below.